<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="receiveBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;Debugger V2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="conectionState">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;State:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Code to request:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="signalSelector">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Clock()</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IF: Instruction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IF: Iadd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: aluOp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: isJump</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: isNotConditional</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: IsEqual</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: memWrite</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: wbi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: memRead</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: aluSrc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: reg1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: reg2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: extendedInstr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: regAddr1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: regAddr2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ID: regDst</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: is_jump_o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: branch_eq_o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: branch_inc_o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: zero</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: jump_address</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: wbi_o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: M_o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: regAddr_o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: data_b_o</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>EXE: out</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MEM: dataFromMem</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MEM: dataFromImm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MEM: regAddrOut</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MEM: wbi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WB: dataToRegFile</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WB: WeRegFile</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rg: registro 0</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="requestButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Request</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendClockButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Clock</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ComEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>COM1</string>
    </property>
   </widget>
   <widget class="QLabel" name="IaddLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="InstructionLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Iadd: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Instruction:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contExe">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start continue execution</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="RegisterViewer">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>70</y>
      <width>191</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Registros:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
